docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc

```diff
@@ -3988,6 +3988,137 @@ In order to use the WIPS feature in PacketFence, please follow those simple step
+
+  aaa profile "packetfence-externalportal"
+  initial-role packetfence-portal
+  radius-interim-accounting
+  radius-accounting "packetfence"
+  rfc-3576-server "192.168.1.5"
+
+Define a policy to permit the traffic.
+
+First add a destination, `Advanced Services > Stateful Firewall > Destinations`:
+
+  netdestination packetfence-portal
+  host 192.168.1.5
+
+Create an ACL for the redirection, `Security > Firewall Policies`:
+
+  Source NAT on VLAN
+  ip access-list session "packetfence-externalportal"
+  alias "user" alias "packetfence-portal" "svc-http" permit queue low
+  alias "user" alias "packetfence-portal" "svc-https" permit queue low
+
+  Source NAT per Application
+  ip access-list session "packetfence-externalportal"
+  alias "user" alias "packetfence-portal" "svc-http" src-nat queue low
+  alias "user" alias "packetfence-portal" "svc-https" src-nat queue low
+
+Now add the newly created policy to the Captive Portal Profile, `Security > User Roles`:
+
+image::docs/images/aruba-config-user-role.png[scaledwidth="100%",alt="User Role"]
+image::docs/images/aruba-interface-redirection.png[scaledwidth="100%",alt="Interface de redirection"]
+
+  user-role "packetfence-portal"
+  access-list session "packetfence-externalportal" position 1
+  access-list session "captiveportal" position 2
+  access-list session "guest-logon-access" position 3
+  access-list session "block-internal-access" position 4
+  access-list session "v6-logon-control" position 5
+  access-list session "captiveportal6" position 6
+  captive-portal "packetfence-externalportal"
+
+Finaly create the SSID and associate the profile to it, `Virtual AP profile`:
+
+  wlan virtual-ap "packetfence-externalportal"
+  ssid-profile "packetfence-externalportal"
+  aaa-profile "packetfence"
+
+General AP settings and master-slave controller case.
+
+image::docs/images/aruba-ap-config1.png[scaledwidth="100%",alt="Config AP 1"]
+image::docs/images/aruba-ap-config2.png[scaledwidth="100%",alt="Config AP 2"]
+image::docs/images/aruba-ap-config3.png[scaledwidth="100%",alt="Config AP 3"]
+image::docs/images/aruba-ap-config4.png[scaledwidth="100%",alt="Config AP 4"]
+image::docs/images/aruba-ap-config5.png[scaledwidth="100%",alt="Config AP 5"]
+image::docs/images/aruba-ap-config6.png[scaledwidth="100%",alt="Config AP 6"]
+image::docs/images/aruba-ap-config7.png[scaledwidth="100%",alt="Config AP 7"]
+image::docs/images/aruba-ap-config8.png[scaledwidth="100%",alt="Config AP 8"]
+image::docs/images/aruba-ap-config9.png[scaledwidth="100%",alt="Config AP 9"]
+image::docs/images/aruba-ap-config10.png[scaledwidth="100%",alt="Config AP 10"]
+image::docs/images/aruba-ap-config11.png[scaledwidth="100%",alt="Config AP 11"]
+
+image::docs/images/aruba-secondary-controller.png[scaledwidth="100%",alt="Secondary controller configuration"]
+
+The next step will be to configure the Webauth in PacketFence, add the switch with the model choice `Aruba Network`, 
```

Comment:
configure the Webauth in PacketFence -> configure the Aruba WiFi controller for WebAuth in PacketFence